src/browser/documentation/help/grant.jsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2002-2019 "Neo4j,"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react'
+import ManualLink from 'browser-components/ManualLink'
+import AdminOnSystemDb from './partials/admin-on-systemdb'
+const title = 'GRANT'
+const subtitle = 'Grant privileges to roles'
+const category = 'security'
+const content = (
+  <>
+    <p>
+      The <code>GRANT</code> command allows an administrator to grant a
+      privilege to a role in order to access an entity.
```

Comment:
entity seems vague. I assume its in the documentation like this too.

Reply:
Yeah, this is coming from the documentation. From my understanding it's vague on purpose, because entity in this context includes many different things. It can be different types of operations on a database, a type of query, or even a type of query applied to a specific type of node/relationship. 

I do agree that is not so clear though. I think `access` is not really appropriate, as it's not just that, but other types of operations as well: traverse/read/write/match in the case of graphs, managing indexes&constraints, and other administration operations in the case of databases.

Reply:
Yeah it sounds vague but its not one I can really comment on much more. I guess if you have taken it from the documentation then at least its consistent.